### PR TITLE
add leaflet-vector-layers v1.6.0

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6746 
@pvnr0082t  please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/JasonSanford/leaflet-vector-layers
 * Official website (optional, not the repository): https://jasonsanford.github.io/leaflet-vector-layers
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 196
   - Count of watchers: 24
   - Count of forks: 54
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

git auto-update config added, cc @JasonSanford

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Didn't mention related issue in commit message, again.

**Comment by kennynaoh** [comment]
Sorry about that, again.
@pvnr0082t 
I have updated the commit message.
Please help me review it again.
Thanks.

File: ajax/libs/leaflet-vector-layers/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/JasonSanford/leaflet-vector-layers.git"
+  },
+  "version": "1.6.0",
+  "description": "Allows to easily create vector layers from a number of geo web services, such as ArcGIS Server, Arc2Earth, GeoIQ, CartoDB and GIS Cloud. ",
+  "author": " JasonSanford",
+  "license": "ISC",
```
Comment (pvnr0082t) [inline ajax/libs/leaflet-vector-layers/package.json]: @kennynaoh How do you know the license is ```ISC```? The author doesn't confirm with you in [issue](https://github.com/JasonSanford/leaflet-vector-layers/issues/63).

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have confirmed with author and updated the license.
Please help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
> I have confirmed with author and updated the license.

@kennynaoh, next time please give us a link to the reference about the author's confirmation, and directly tell us the license in the comment, so that the review process can be much more easier and faster. I saw the issue about the confirming the license, maybe it can be more polite, though I know the problem may comes from the English skill...

**Comment by kennynaoh** [comment]
Sorry about that!
I will be more careful and more polite next time.
Thanks for yours help.

**Pull request merged**